docs/compatibility.md

```diff
@@ -0,0 +1,75 @@
+# Registry Compatibility
+
+
+## Synopsis
+*If a Manifest is pulled by _Digest_ from a Registry 2.3 with Docker Engine 1.9
+and older, and the Manifest was pushed with Docker Engine 1.10, a security check
+will cause the Engine to receive a Manifest it cannot use and the pull will fail.*
+
+## Registry Manifest Support
+
+
+Historically, the Registry has supported a [single Manifest type](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md)
+known as _Schema 2 Version 1_.
```

Comment:
Isn't this schema 1 version 1? We use schema2 to talk about the new manifest format.